content/sensu-go/5.19/reference/backend.md

```diff
@@ -1291,7 +1297,21 @@ Here are some log rotate sample configurations:
 }
 {{< /highlight >}}
 
-#### sysvinit
+Without the `postrotate` script, the backend will not reload.
+This will cause sensu-backend (and sensu-agent, if translated for the Sensu agent) to no longer write to the log file, even if logrotate recreates the log file.
+
+In this script, `systemctl reload` sends a _SIGHUP_ signal to the sensu-backend process.
+
+- In Sensu Go versions 5.19.2 and earlier, this reload causes sensu-backend to fully restart.
+If you are running a clustered backend, all cluster members will restart at the same time.
+- In [Sensu Go version 5.19.3][30] and later, the _SIGHUP_ signal causes only event logging components to reload, so cluster members will not simultaneously restart.
```

Comment:
From reading over https://github.com/sensu/sensu-go/pull/3723 I think the entire `backend` component is reloaded, not just event logging.

```suggestion
- In [Sensu Go version 5.19.3][30] and later, the _SIGHUP_ signal causes the `backend` component to reload, instead of restarting the process.
```

That said, we added a handler for SIGUSR1 in 5.20.0. I think we'd be well served to add a section in this doc which describes signal handling and then refer to that section from this log rotate content as needed.

Reply:
@cwjohnston I am thinking that the best way to do this is:
1. Propagate the changes in this PR to all versions (with minor adjustments based on versions) and merge
2. Create a new issue to explain the signal handler (https://github.com/sensu/sensu-go/pull/3723) in the backend reference and add a reference link in the 5.20 docs.

What do you think?